### PR TITLE
Add benchmark module

### PR DESCRIPTION
**Pull request by csh519**
Add benchmark module and modify `README`.

### CODE REVIEW

**Review by yuzzyuzz — DISMISSED** [review]
Thanks for adding benchmark module!

**Review by meanrin — COMMENTED** [review]
Please check comment below
Other than that - LGTM

File: README.md [hunk 1]
```diff
@@ -270,20 +274,47 @@ It may be difficult to check the license of an existing project due to obfuscati
...
 Name | TP | FP | TN | FN | FPR | FNR | Precision | Recall | F1
 ---- | -- | -- | -- | -- | --- | --- | --------- | ------ | --
-ours (to be released) | 4,231 | 1,592 | 52,511 | 1,771 | 0.0294 | 0.29506 | 0.7266 | 0.7049 | 0.7156
-detect-secrets | 2,862 | 10,467 | 44,508 | 3,140 | 0.1903 | 0.5231 | 0.2147 | 0.4768 | 0.2961
-gitleaks | 1,064 | 1,068 | 52,838 | 4,938 | 0.0198 | 0.8227 | 0.49906 | 0.1772 | 0.2616
-shhgit | 324 | 277 | 53,629 | 5,678 | 0.0051 | 0.94601 | 0.5391 | 0.0539 | 0.0981
-truffleHog | 1,756 | 129,343 | 41,622 | 4,246 | 0.7565 | 0.7074 | 0.0133 | 0.2925 | 0.0256
-credential-digger | 637 | 25,532 | 49,997 | 5,365 | 0.33804 | 0.8938 | 0.0243 | 0.1061 | 0.0396
-wraith(gitrob) | 1,504 | 3,062 | 52,149 | 4,498 | 0.0554 | 0.7494 | 0.3293 | 0.2505 | 0.2846
+**CredSweeper** | 4,787 | 573 | 19,452,707 | 1,215 | 0.0000294552 | 0.2024325225 | 0.8930970149 | 0.7975674775 | 0.8426333392
```
[the remark this answers is not in the document]
Comment (csh519) [reply README.md:311]: I added notice by 64cd55e. Please check.

**Review by silentearth — APPROVED** [review]
Thank you for updating.
I hope that people test their tools or other opensource tools.

**Pull request merged**